sqlmesh/core/model/definition.py

```diff
@@ -2453,7 +2453,7 @@ def _create_model(
             statements.extend(property_values.expressions)
 
     jinja_macro_references, used_variables = extract_macro_references_and_variables(
-        *(gen(e) for e in statements)
+        *(gen(e if isinstance(e, exp.Expression) else e[0]) for e in statements)
```

Comment:
Are we sure this is a non-breaking change?

Reply:
It should be fine, I think. These `gen`erated strings are used for extracting macro & variable references. If I'm not mistaken, we'd previously produce `"(True, <expression SQL>)"` due to reaching [this](https://github.com/tobymao/sqlglot/blob/main/sqlglot/optimizer/simplify.py#L1371-L1373) section. After this PR we'll only search `<expression SQL>`.